lib/_tls_common.js

```diff
@@ -27,22 +27,10 @@ const {
 
 const tls = require('tls');
 
-const {
-  codes: {
-    ERR_TLS_INVALID_PROTOCOL_VERSION,
```

Comment:
Destructing symbols and constants should be safe and disallowing this could be way too verbose.